### ISSUE TITLE
Regarding the attention map

### ISSUE THREAD

**Issue by gemcollector** [MishaLaskin/rad#16]
I noticed that you drew an attention map in the text. How do you draw an attention map? Can you provide the source code?

**Comment by MishaLaskin** [comment]
The attention maps are just activations of one of the CNN layers. To make
them - pick a CNN layer run a softmax over each channel and take the mean.

**Comment by longfeizhang617** [comment]
> I noticed that you drew an attention map in the text. How do you draw an attention map? Can you provide the source code?

I have noticed the attention map too, have you solved this issue?

**Comment by Dongzhou-1996** [comment]
> > I noticed that you drew an attention map in the text. How do you draw an attention map? Can you provide the source code?
> 
> I have noticed the attention map too, have you solved this issue?

The author mentioned that their method is similar to the algorithm proposed in [PAYING MORE ATTENTION TO ATTENTION:
IMPROVING THE PERFORMANCE OF CONVOLUTIONAL NEURAL NETWORKS VIA ATTENTION TRANSFER](https://openreview.net/forum?id=Sks9_ajex), which simply sums the activation tensor of any convolutional layer along the channel dimension (i.e. a tensor in [BxCxWxH] -> attention map [BxWxH]). Corresponding codes can be seen in this [link](https://github.com/szagoruyko/attention-transfer/blob/master/visualize-attention.ipynb). The only difference between these two methods is the spatial softmax operation, I think it is added by Laskin to show the attention map better and highlight the fixation of a convolutional layer.
